common/src/state/action.rs

```diff
@@ -103,8 +103,8 @@ pub enum Action {
     CancelRequest(Identity),
 
     /// Handle a new incoming friend request
-    #[display(fmt = "IncomingRequest")]
-    IncomingRequest(Identity),
+    // #[display(fmt = "IncomingRequest")]
+    // IncomingRequest(Identity),
```

Comment:
why comment this out? Should we just remove it if it's dead code?

Reply:
i didn't know if something was planned for later, but I'm fine with deleting it.

Reply:
Let's nuke it 👍